models/yang/annotations/openconfig-network-instance-annot.yang

```diff
@@ -18,7 +18,7 @@ module openconfig-network-instance-annot {
 
     deviation /oc-netinst:network-instances/oc-netinst:network-instance/oc-netinst:name {
       deviate add {
-          sonic-ext:field-transformer "network_instance_name_field_xfmr";
+          sonic-ext:field-transformer "network_instance_name_key_xfmr";
```

Comment:
Originally the field-transformer doesn't have the "key" in its name because there is already a key transformer "network_instance_table_key_xfmr"